main/auth/twitch.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+
+import flask
+
+import auth
+import config
+import model
+import util
+
+from main import app
+
+twitch_config = dict(
+  access_token_method='POST',
+  access_token_url='https://id.twitch.tv/oauth2/token',
+  api_base_url='https://api.twitch.tv/helix/',
+  authorize_url='https://id.twitch.tv/oauth2/authorize',
+  client_id=config.CONFIG_DB.twitch_client_id,
+  client_secret=config.CONFIG_DB.twitch_client_secret,
+  request_token_params={
```

Comment:
I think this `request_token_params` should be `client_kwargs`.

Reply:
That worked.. and now we have a new error!